openedx/core/djangoapps/schedules/models.py

```diff
@@ -5,10 +5,18 @@
 
 class Schedule(TimeStampedModel):
     enrollment = models.OneToOneField('student.CourseEnrollment', null=False)
-    active = models.BooleanField(default=True, help_text=_('Indicates if this schedule is actively used'))
-    start = models.DateTimeField(help_text=_('Date this schedule went into effect'))
+    active = models.BooleanField(
+        db_index=True,
```

Comment:
What queries do we anticipate running? We know we will want to get all schedules with deadlines between a certain range. Would we ever want to get schedules based on when the schedule went into effect? Do we ever anticipate the active flag being set to `False`?